### ISSUE TITLE
fix(StripFrontmatter): preserve trailing blank line for frontmatter-only files

### ISSUE THREAD

**Issue by shinagawa-web** [shinagawa-web/gomarklint#162]
## Overview

`StripFrontmatter` consumes **all** blank lines immediately after the closing `---`, including the file's trailing newline. For frontmatter-only files this leaves an empty string as the stripped body, which causes `final-blank-line` to report a false positive.

## Problem

`docs/content/docs/_index.md` is a valid frontmatter-only file:

```markdown
---
title: "Docs"
weight: 1
---
```

`StripFrontmatter` processes it as follows:

```
lines = ["---", "title: \"Docs\"", "weight: 1", "---", ""]
                                                              ^^^^
                                         trailing blank line (the final \n)
```

The skip loop advances past the trailing `""` and returns `""` with offset 5. `strings.Split("", "\n")` produces `[""]`, and `CheckFinalBlankLine` flags it because `len < 2`.

## Expected behavior

A frontmatter-only file that ends with `\n` should pass `final-blank-line`. The trailing blank line belongs to the file, not to the inter-frontmatter gap.

## Root cause

`internal/file/reader.go` — `StripFrontmatter`:

```go
skip := i + 1
for skip < len(lines) && strings.TrimSpace(lines[skip]) == "" {
    skip++   // consumes ALL trailing blank lines unconditionally
}
return strings.Join(lines[skip:], "\n"), skip
```

When no non-empty content follows the frontmatter the loop overshoots, leaving an empty body.

## Proposed fix

Stop the skip loop when only trailing empty lines remain (i.e. there is no subsequent non-empty line). If the loop would consume everything, return `"\n"` to represent an effectively empty body that still ends with a blank line.

## References

- Introduced as a side-effect while extending the CI lint step to cover `docs/` (#157)
